### PR TITLE
v0.9.10-rc3 - Updated weights and transaction_version

### PR DESCRIPTION
**Pull request by s3krit**
Weights were updated in the following jobs, with patches downloaded and applied:

* [polkadot](https://gitlab.parity.io/parity/polkadot/-/jobs/1119134)
* [kusama](https://gitlab.parity.io/parity/polkadot/-/jobs/1119135)
* [westend](https://gitlab.parity.io/parity/polkadot/-/jobs/1119136)

Transaction versions were updated due to this change: #3693  - Note that I updated the transaction version for all three runtimes - I'm not sure if that is required for polkadot.

### CODE REVIEW

**Review by chevdor — APPROVED** [review]
Looking fine besides the comments I made about a few big changes.

File: runtime/polkadot/src/weights/pallet_elections_phragmen.rs [hunk 1]
```diff
@@ -143,12 +143,10 @@ impl<T: frame_system::Config> pallet_elections_phragmen::WeightInfo for WeightIn
 	// Storage: PhragmenElection Candidates (r:1 w:0)
 	// Storage: Balances Locks (r:250 w:250)
 	// Storage: System Account (r:250 w:250)
-	fn clean_defunct_voters(v: u32, d: u32, ) -> Weight {
+	fn clean_defunct_voters(v: u32, _d: u32, ) -> Weight {
```
[the remark this answers is not in the document]
Comment (kianenigma) [reply runtime/polkadot/src/weights/pallet_elections_phragmen.rs:146]: This tx has not changed in ages, something in the benchmarking setup might have changed.

[the remark this answers is not in the document]
Comment (chevdor) [reply runtime/polkadot/src/weights/pallet_elections_phragmen.rs:146]: The change is legit anyway.

File: runtime/polkadot/src/weights/pallet_elections_phragmen.rs [hunk 2]
```diff
@@ -143,12 +143,10 @@ impl<T: frame_system::Config> pallet_elections_phragmen::WeightInfo for WeightIn
...
 		(0 as Weight)
-			// Standard Error: 55_000
-			.saturating_add((108_480_000 as Weight).saturating_mul(v as Weight))
 			// Standard Error: 53_000
-			.saturating_add((363_000 as Weight).saturating_mul(d as Weight))
+			.saturating_add((105_308_000 as Weight).saturating_mul(v as Weight))
```
[the remark this answers is not in the document]
Comment (chevdor) [reply runtime/polkadot/src/weights/pallet_elections_phragmen.rs]: The [new version](https://github.com/paritytech/polkadot/pull/3865/files#diff-0f1527c46e323527b66808e6e920f7da4647d0c27563a6309403177091dbd61dR151) looks better

File: runtime/polkadot/src/weights/pallet_identity.rs [hunk 3]
```diff
@@ -89,70 +89,70 @@ impl<T: frame_system::Config> pallet_identity::WeightInfo for WeightInfo<T> {
 	// Storage: Identity IdentityOf (r:1 w:1)
 	// Storage: Identity SuperOf (r:0 w:100)
 	fn clear_identity(r: u32, s: u32, x: u32, ) -> Weight {
-		(47_890_000 as Weight)
-			// Standard Error: 12_000
-			.saturating_add((141_000 as Weight).saturating_mul(r as Weight))
+		(50_788_000 as Weight)
+			// Standard Error: 13_000
+			.saturating_add((32_000 as Weight).saturating_mul(r as Weight))
```
[the remark this answers is not in the document]
Comment (chevdor) [reply runtime/polkadot/src/weights/pallet_identity.rs]: The new run looks better

**Pull request merged**